<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIMainClass</class>
 <widget class="QMainWindow" name="UIMainClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ZheatEngine</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="9" column="2" colspan="3">
     <widget class="QGroupBox" name="gruopOption">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>Memory Scan Options</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditStop">
         <property name="text">
          <string>7FFFFFFFFFFFFFFF</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditStart">
         <property name="text">
          <string>0000000000000000</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QCheckBox" name="checkBoxPause">
         <property name="text">
          <string>Pause the game while searching</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="spinBoxAlignment">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>4096</number>
         </property>
         <property name="value">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="checkBoxAlignment">
         <property name="text">
          <string>Aligned Only</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" rowspan="2">
     <widget class="QPushButton" name="buttonOpen">
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2" colspan="3">
     <widget class="QGroupBox" name="groupParam">
      <property name="title">
       <string>Scan Parameters</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </item>
    <item row="1" column="1" colspan="4">
     <widget class="QProgressBar" name="progressBar">
      <property name="maximum">
       <number>1</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Value Type</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Scan Type</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="buttonNextScan">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Next Scan</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="3">
     <widget class="QLabel" name="labelProcessInfo">
      <property name="text">
       <string>No Process Selected</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="labelIcon">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelFound">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QComboBox" name="comboBoxValueType"/>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="buttonNewScan">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>First Scan</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="10" column="4">
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>Add Address Manually</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QComboBox" name="comboBoxScanType"/>
    </item>
    <item row="5" column="4" rowspan="3">
     <widget class="QOpenGLWidget" name="openGLWidget"/>
    </item>
    <item row="5" column="0" rowspan="5" colspan="2">
     <widget class="QTreeWidget" name="treeWidgetResults">
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="10" column="0" colspan="4">
     <widget class="QTreeWidget" name="treeWidgetWatchList">
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>852</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ZheatEngine.qrc"/>
 </resources>
 <connections/>
</ui>
